b2sdk/exception.py

```diff
@@ -438,6 +452,11 @@ def interpret_b2_error(status, code, message, response_headers, post_params=None
         return MissingPart(post_params.get('fileId'))
     elif status == 400 and code == "part_sha1_mismatch":
         return PartSha1Mismatch(post_params.get('fileId'))
+    elif status == 400 and code == "bad_request":
+        matcher = UPLOAD_TOKEN_USED_CONCURRENTLY_ERROR_MESSAGE_RE.match(message)
+        if matcher is not None:
```

Comment:
if matcher _is_ None, then a 400 `bad_request` error is going to be silently swallowed by that `if` branch!